plugins/action/zos_copy.py

```diff
@@ -154,7 +152,7 @@ def run(self, tmp=None, task_vars=None):
             else:
                 if is_src_dir:
                     path, dirs, files = next(os.walk(src))
-                    if dirs:
+                    if not is_uss and dirs:
                         result["msg"] = "Subdirectory found inside source directory"
```

Comment:
Unrelated to the PR but thinking we could improve this message, basically the code does not support if the source is a directory with more directories and being written into a data set. Writing to another dir is fine and supported.

"When source is a directory with sub directories,  the destination must be another directory not a data set" <---- maybe something like that?

Reply:
We can update the message while we're already reviewing the PR. Using your suggestion, maybe we could make the fact that we hit an error more explicit? Something like this: "Cannot copy a source directory with sub directories to a data set, the destination must be another directory"

Reply:
Perfect "Cannot copy a source directory with sub directories to a data set, the destination must be another directory", just thought since we are here now why not make the change,  I appreciate the details (users will too :) )